templates/webapi/test/link_context.html.ep

```diff
@@ -0,0 +1,23 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+<div class="container">
+        <p>
+            Test-Module:
+            
+                <tt><%= $contextpath %></tt>
```

Comment:
The tt element is obsolete. Use CSS instead. It would also be nice to fix the place where you copied that from and avoid the duplication.

Reply:
i will do

Reply:
s/<tt>/<code>